tf2_ros_py/test/test_buffer.py

```diff
@@ -146,3 +147,9 @@ def test_buffer_non_default_cache(self):
         assert transform.transform.translation.x == output.transform.translation.x
         assert transform.transform.translation.y == output.transform.translation.y
         assert transform.transform.translation.z == output.transform.translation.z
+
+
+pytestmark = pytest.mark.skipif(
```

Comment:
This isn't actually being used anywhere, so it wouldn't have any impact on the build.